network_test.go

```diff
@@ -0,0 +1,111 @@
+//
+// Copyright (c) 2017 Intel Corporation
```

Comment:
Nit: 2018.

Reply:
Ah, yes, happy new year to you as well.  Will update.